<launch>
  <arg name="device" default="/dev/ttyCTI3" />

  <node pkg="hsrb_power_ecu" name="power_ecu_main_loop" exec="hsrb_power_ecu_main" >
    <param name="port_name" value="$(var device)" />
    <param name="port_receive_timeout_ms" value="0.5" />
    <param name="battery_state/publish_rate" value="1.0" />
    <param name="battery_charging/publish_rate" value="10.0" />
    <param name="base_f_bumper_sensor/publish_rate" value="10.0" />
    <param name="base_b_bumper_sensor/publish_rate" value="10.0" />
    <param name="emergency_stop_button/publish_rate" value="10.0" />
    <param name="wireless_stop_button/publish_rate" value="10.0" />
    <param name="wireless_stop_enable/publish_rate" value="10.0" />
    <param name="runstop_button/publish_rate" value="10.0" />
  </node>
</launch>
